modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -5,6 +5,11 @@
 [id="configuring-host-network-interfaces-in-the-install-config.yaml-file_{context}"]
 = (Optional) Configuring host network interfaces in the `install-config.yaml` file
 
+[IMPORTANT]
```

Comment:
You can't put a link to either docs.openshift.com or an OpenShift collection on access.redhat.com anywhere in this docs set. A user must stay in the same docs experience that they started from.

Also, you can't put a note directly after the H1. The text after an H1 can only be a short paragraph.

Reply:
Updated as suggested.